assets/javascripts/discourse/admin-discourse-ai-plugin-route-map.js

```diff
@@ -6,17 +6,17 @@ export default {
   map() {
     this.route("discourse-ai-personas", { path: "ai-personas" }, function () {
       this.route("new");
-      this.route("show", { path: "/:id" });
+      this.route("edit", { path: "/:id" });
```

Comment:
Can you also make the paths `/:id/edit`? Also something Ted and I discussed in passing, I think it's probably good to standardize